src/enums.h

```diff
@@ -61,6 +61,14 @@ enum itemAttrTypes : uint32_t {
 	ITEM_ATTRIBUTE_DOORID = 1 << 22,
 };
 
+enum customAttrTypes : uint8_t {
```

Comment:
I think the convention in tfs is to use CamelCase for type names and camelBack for object/variable/function/member names.